charts/airbyte/templates/secret.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if eq .Values.global.deploymentMode "oss"  }}
+{{- if false }}
```

Comment:
Is there a point to the template if `if false` is always evaluated as false?

Reply:
Left this here as a template for the k8s secret that needs to be created before Airbyte is deployed. Prior to this, the deployment would create the secret using hard-coded values from the `values.yaml` file.

Reply:
if deploymentMode is smth used for other stuff too, you could use a different global variable like `useExistingSecrets` or smth like that ... and here you could check both conditions..if depmode=oss and useexistingsecrets=false